### PR TITLE
[Bugfix beta] Settable layout cp

### PR DESCRIPTION
**Pull request by stefanpenner**
depends on: #10920 will rebase, when that lands.

Also I will add a separate PR for beta branch, as the syntax is different.

### CODE REVIEW

**Comment by stefanpenner** [comment]
travis claims this is actually green: https://travis-ci.org/emberjs/ember.js/builds/59537515 ...

File: packages/ember-views/lib/views/view.js [hunk 1]
```diff
@@ -763,14 +763,20 @@ var View = CoreView.extend(
 
     @property layout
     @type Function
-  */
-  layout: computed('layoutName', function(key) {
-    var layoutName = get(this, 'layoutName');
-    var layout = this.templateForName(layoutName, 'layout');
+    */
+  layout: computed('layoutName', {
```
Comment (rwjblue) [inline packages/ember-views/lib/views/view.js]: Shouldn't we also add `defaultLayout` to the dependent keys?

Comment (stefanpenner) [inline packages/ember-views/lib/views/view.js]: i don't believe so. I am not sure why `defaultLayout` still exists, is there some piece of functionality I am missing?

**Comment by stefanpenner** [comment]
Travis appears to be [green](https://travis-ci.org/emberjs/ember.js/builds/59537515) but here it is reported red. I suspect this is good to go.

File: packages/ember-views/lib/views/view.js (hunk 1 above)
Comment (rwjblue) [inline packages/ember-views/lib/views/view.js]: Sounds good, thank you for clarifying. :smile:

**Pull request merged**